sybil/parsers/markdown/skip.py

```diff
@@ -8,6 +8,7 @@ class SkipParser(AbstractSkipParser):
     """
 
     def __init__(self) -> None:
-        super().__init__([
-            DirectiveInHTMLCommentLexer('skip'),
-        ])
+        super().__init__(
```

Comment:
I chose to switch to keyword arguments because as a reader I wouldn't necessarily know that the second argument is the directive to use. I'm not hard line on this!